### PR TITLE
[onert] Support Mul int8 quantization

### PR DESCRIPTION
**Pull request by glistening**
Mul supports both of int8 and uint8 quantization.

ONE-DCO-1.0-Signed-off-by: Sanggyu Lee <sg5.lee@samsung.com>

### CODE REVIEW

**Comment by glistening** [comment]
@nnfw-bot test format-checker

Strange. I did not change `compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp`.

File: compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp [hunk 1]
```diff
@@ -100,7 +100,7 @@ luci::CircleTranspose *create_pre_transpose(luci::CircleNode *node)
 uint32_t cal_offset(const loco::TensorShape &dimension, const uint32_t *indices)
 {
   return indices[0] * dimension.dim(1).value() * dimension.dim(2).value() *
-             dimension.dim(3).value() +
+           dimension.dim(3).value() +
```
Comment (glistening) [inline compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp:103]: This change is not mine. However, format-checker want to fix this format error in my PR.

File: compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp [hunk 2]
```diff
@@ -143,7 +143,7 @@ luci::CircleConst *create_NHWC_from_NCHW(luci::CircleConst *constant)
           uint32_t nchw_indices[4] = {n, c, h, w};
           uint32_t nhwc_indices[4] = {n, h, w, c};
           auto data =
-              constant->at<loco::DataType::FLOAT32>(cal_offset(nchw_dimension, nchw_indices));
+            constant->at<loco::DataType::FLOAT32>(cal_offset(nchw_dimension, nchw_indices));
```
Comment (glistening) [inline compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp:146]: Same

File: compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp (hunk 1 above)
Comment (glistening) [reply compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp:103]: #5496 is created. I will remove this change in my PR soon.

File: tests/nnfw_api/src/one_op_tests/Mul.cc [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  int out = cgen.addTensor({{1, 2, 3, 1}, circle::TensorType::TensorType_FLOAT32});
+  cgen.addOperatorMul({{lhs, rhs}, {out}}, circle::ActivationFunctionType_NONE);
+  cgen.setInputsAndOutputs({lhs, rhs}, {out});
+
+  _context = std::make_unique<GenModelTestContext>(cgen.finish());
+  _context->setBackends({"acl_cl", "acl_neon", "cpu"});
+  _context->expectFailCompile();
+
+  SUCCEED();
+}
```
Comment (wateret) [inline tests/nnfw_api/src/one_op_tests/Mul.cc:115]: It looks like there are 4 positive cases and 2 negatives. Could you make 2 or more negative cases?

Comment (glistening) [reply tests/nnfw_api/src/one_op_tests/Mul.cc:115]: @wateret Sure, I've added.

**Review by wateret — APPROVED** [review]
LGTM

**Review by hseok-oh — APPROVED** [review]

**Review by ragmani — APPROVED** [review]

**Pull request merged**